specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -805,6 +907,56 @@
             "create",
             "read"
           ]
+        },
+        "additionalCapabilities": {
+          "type": "array",
+          "description": "List of additional capabilities to enable or disable for a Disk Pool",
+          "items": {
+            "$ref": "#/definitions/AdditionalCapability"
+          }
+        }
+      }
+    },
+    "DiskPoolUpdateProperties": {
+      "type": "object",
+      "description": "Properties section of the Disk Pool request payload.",
+      "properties": {
+        "provisioningState": {
+          "description": "State of the operation on the resource",
+          "type": "string",
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Invalid",
+            "Unknown",
+            "Healthy",
+            "Unhealthy"
+          ],
+          "x-ms-enum": {
+            "name": "OperationalStatus",
+            "modelAsString": true
+          },
+          "description": "Operational status of the Disk Pool"
+        },
+        "disks": {
```

Comment:
Per our before discussion, all properties can be update should be supported in diskpool update, but currently seems only disks, and additionalCapabilities. Please clarify.

Reply:
disks, additionalCapabilities and tags are the only mutable properties of a diskPool

other properties such as subnetId, availabilityZones, name, location cannot be updated